src/target/llvm/codegen_hexagon.cc

```diff
@@ -69,6 +69,9 @@ class CodeGenHexagon final : public CodeGenLLVM {
   llvm::Module* GetModulePtr() const { return module_.get(); }
 
  protected:
+  void CreatePrintf(const std::string& format,
+                    const llvm::ArrayRef<llvm::Value*>& format_args) final;
```

Comment:
`llvm::ArrayRef` is meant to be passed by value, sorry, I should have been clearer about it (it's an array "view", like `std::string_view`).  This can be fixed in a different PR though.